.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+### Contributing to ZFS on Linux
+
+<p align="center"><img src="http://zfsonlinux.org/images/zfs-linux.png"/></p>
+
+*First of all, thank you for taking the time to contribute!*
+
+By using the following guidelines, you can help us make ZFS on Linux even better.
+
+#### Table Of Contents
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+
+  * [Get ZFS](#get-zfs)
+  * [Debugging ZFS](#debugging-zfs)
+  * [Where can I ask for help?](#where-can-I-ask-for-help)
+  
+[How Can I Contribute?](#how-can-i-contribute)
+
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+  * [Testing](#testing)
+  
+[Style Guides](#style-guides)
+ 
+  * [Coding Conventions](#coding-conventions)
+
+Helpful resources
+
+  * [ZFS on Linux wiki](https://github.com/zfsonlinux/zfs/wiki)
+  * [OpenZFS Documentation](http://open-zfs.org/wiki/Developer_resources)
+
+
+## What should I know before I get started?
+
+### Get ZFS
+
+You can build fresh zfs packages using 
```

Comment:
`You can build zfs packages by following these instructions or install stable packages from your distribution's repository.`